### ISSUE TITLE
Focus not visible on CPR training

### ISSUE THREAD

**Issue by jill-iterators** [CityOfBoston/Iterators#143]
Screengrab/videos (insert file names here)
2020, Focus not visible on CPR training.mp4

Link
https://d8-stg.boston.gov/departments/emergency-medical-services/cpr-training

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.5 > Chrome 84

Trusted Tester Test
n/a

Brief Description
When testing accessibility, keyboard jumps to a new page or area and Focus is lost. 

Steps to reproduce
1)	00:00, Go to the linked page, link is noted above.
2)	00:22, Note: Tabbing through the link to Coronavirus Updates, then Department Information. The next tab should be the three-tab interfaces, 
3)	00:26, Pressing tab key jumps to section titled: Learn About Hands Only CPR. User has lost Focus, unsure where cursor is. 
4)	00:34, Using the Shift+tab key, jump back for last placement of cursor, which is Department Information. Retesting confirms loss of Focus due to no visibility of cursor.

Desired Result
•	Able to visually see the cursor at all times while tabbing trough the linked page on website.

Actual Result
•	Unable to visually see the cursor at all times while tabbing trough the linked page on website.

**Comment by subaha-cob** [comment]
Test on: https://d8-uat.boston.gov/

**Comment by andrebtlima** [comment]
@subaha-cob I'm still unable to navigate to the How To tabs on UAT

**Comment by subaha-cob** [comment]
@andrebtlima this works for me. Please make sure you are on UAT and clear your cache or open incognito window.

**Comment by jill-iterators** [comment]
This is working now for me. Will close out matter.
